Gemfile

```
@@ -19,3 +19,4 @@ gem 'responders'
 gem 'rspec'
 gem 'rubocop'
 gem 'yard'
+gem 'sqlite3',           '~>1.4.0'
```

For `~>`, we always want exactly one decimal place, so it works as SemVer-compatible.
```suggestion
gem 'sqlite3',           '~> 1.4'
```